### ISSUE TITLE
Elbow extension contracture

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7693]
**Preferred term label:**
Elbow extension contracture

**Synonyms**

**Definition (free text, please give PubMed ID)**
An elbow contracture that limits the ability of the elbow joint to flex (flexion of the the elbow joint refers to bending the elbow joint to bring the hand closer to the shoulder), meaning that the elbow is fixed in an extended (straight) position.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
